packages/bpk-component-barchart/src/BpkChartAxis.js

```diff
@@ -76,7 +78,25 @@ const getAxisConfig = ({ orientation, margin, height, width, scale }) => {
   };
 };
 
-const BpkChartAxis = props => {
+type Props = {
+  height: number,
+  width: number,
+  margin: {
+    top: number,
+    bottom: number,
+    left: number,
+    right: number,
+  },
+  scale: Function,
```

Comment:
We typically try to avoid `Function` in favour of more specific syntax like `(argType) => returnType`, is there something more specific we can use here?